similarity/config/configurations.properties

```diff
@@ -0,0 +1,20 @@
+wordMetric	paragram
```

Comment:
Wait are you suggesting dropping `SimConfigurator` or `configurations.properties`?